Testscripts/Windows/GPU-DRIVER-INSTALL.ps1

```diff
@@ -29,13 +29,27 @@ function Start-Validation {
     # region PCI Express pass-through in lsvmbus
     $PCIExpress = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
         -username $superuser -password $password "lsvmbus -vv" -ignoreLinuxExitCode
+    if ( $PCIExpress ) {
+        Write-Debug "Successfully fetched the PCIExpress result from lsvmbus"
+    } else {
+        Write-Error "Could not succeed to fetch out PCIExpress result from lsvmbus"
+    }
+
     Set-Content -Value $PCIExpress -Path $LogDir\PCI-Express-passthrough.txt -Force
     # Scope to match GPUs only since there can be other pass-through devices
     $pciExpressCount = (Select-String -Path $LogDir\PCI-Express-passthrough.txt -Pattern "Device_ID.*47505500").Matches.Count
+    if ( $pciExpressCount -gt 0 ) {
+        Write-Debug "Successfully fetched more than a PCI Expess device "
+    } else {
+        Write-Error "Could not fetched the PCI Express device count"
```

Comment:
Typo

Reply:
fixed in 9626be3c614ab3d45d8977198984ea0a43941873